libs/langchain/langchain/llms/minimax.py

```diff
@@ -29,8 +30,7 @@ class _MinimaxEndpointClient(BaseModel):
     api_key: str
     api_url: str
 
-    @root_validator(pre=True)
```

Comment:
why remove the root_validator decorator?

Reply:
It was my mistake that we I resolved the conflicts, thanks for reminding me, I will fix it.

Reply:
I've fixed the comment, please help with a review again, thanks! @baskaryan